go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/hashicorp/raft-boltdb
 
-go 1.12
+go 1.16
```

Comment:
[nit]: could we make this a separate PR? unless, it's needed for this PR of course.

Reply:
It is needed to enable using `_` in literal integers.